content/applications/productivity/studio/pdf_reports.rst

```diff
@@ -0,0 +1,340 @@
+===========
+PDF reports
+===========
+
+Odoo uses HTML and CSS technologies to create reports (PDF documents). HTML elements provide the
+basic structure, while CSS interacts with the elements to establish how they are shown to the user.
```

Comment:
> HTML elements provide the basic structure, while CSS interacts with the elements to establish how they are shown to the user
filler text? I don't think user needs to understand that, seems like it's gonna bring more confusion than anything else. I'd limit myself to explaining that reports are basically web pages, with headers/footers slapped on top.

Reply:
Indeed, it's kind of a filler intro text that I took from the previous Studio documentation.
I'm not too sure what I could say here instead. Maybe simply: 

> With Studio, you can edit existing PDF reports (e.g., orders and quotations) or create new ones.